src/components/form/form.stories.js

```diff
@@ -0,0 +1,108 @@
+import { html } from "lit-html";
+import "../form";
+import "../form-grid";
+import "../input-field";
+import "../button";
+import "../form-message";
+import styles from "./styles.scss";
+import inputFieldStyles from "../input-field/styles.scss";
+import formGridStyles from "../form-grid/styles.scss";
+import buttonStyles from "../button/styles.scss";
+import formMessageStyle from "../form-message/styles.scss";
+import { stylesheet, docsIntro } from "../../../.storybook/utils.js";
+import { args, argTypes } from "./config";
+
+export default {
+  title: "native-elements/vl-form",
+  decorators: [
+    (story) =>
+      html`
+        ${stylesheet(
+        `${styles}${inputFieldStyles}${formGridStyles}${buttonStyles}${formMessageStyle}`
+      )}${story()}
+      `,
+  ],
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          stylesheets: ["form"],
+          root: "form",
+          intro: "Form element.",
+        }),
+      },
+    },
+  },
+  args,
+  argTypes,
+};
+
+export const Default = ({ validate }) => html`
+  <div style="max-width: 800px">
+    <p>
+      For demo purposes we have reduced the max width of this component to
+      800px.
+    </p>
+    <form is="vl-form" ?data-vl-validate=${validate}>
```

Comment:
Aangezien de validate (voorlopig) niet reactief is, gingen we deze id stories niet standaard aanzetten en de control disabelen?

Reply:
Fixed. Heb de arg op true gezet en de control disabled.

Reply:
Extra story gemaakt voor Validatie.